README.md

```diff
@@ -1 +1,20 @@
-# Html-Forms
\ No newline at end of file
+# Html-Forms
+In this project, we cloned the Sign Up page for mint.com. We created the logos sections, form area and footer section.
+
+In this project, we created:
+
+1. Sign in section at the top-right part of the page.
+2. Sign up form as the main section of the page. We have three input fields and a create account button. We also have privacy policy and terms of service declaration in a paragraph inside the form area.
+3. Footer section with copyright declaration at the bottom of the page.
+
+![App screenshot](images/mint-screenshot.png)
```

Comment:
# [OPTIONAL] - But I strongly recommend.
Kindly in the screenshot IMG add a dot at the beginning of the URL of the image to be accessible to all. I cant see it from my end when I clone your repo. 

replace this `![App screenshot](images/mint-screenshot.png)` with this  `![App screenshot](./images/mint-screenshot.png)`



This is what I mean: 👇 

![SCREENSHOT](https://user-images.githubusercontent.com/22345961/78132362-4779d480-7425-11ea-8d3b-61a7da76663d.png)